### ISSUE TITLE
[BUG] Deprecated positional argument(s) used in SFTTrainer, please use the SFTConfig to set these arguments instead. How to mitigate this?

### ISSUE THREAD

**Issue by jackswl** [huggingface/autotrain-advanced#728]
### Prerequisites

- [X] I have read the [documentation](https://hf.co/docs/autotrain).
- [X] I have checked other issues for similar problems.

### Backend

Local

### Interface Used

CLI

### CLI Command

```
!autotrain --config path-to.yml
```

```
task: llm-sft
base_model: teknium/OpenHermes-2.5-Mistral-7B
project_name: XXX
log: none
backend: local

data:
  path: /content
  train_split: train
  valid_split: null
  chat_template: null
  column_mapping:
    text_column: text

params:
  block_size: 256
  model_max_length: 512
  epochs: 1
  batch_size: 2
  lr: 3e-5
  peft: true
  quantization: int4
  target_modules: all-linear
  padding: right
  optimizer: adamw_torch
  scheduler: cosine
  gradient_accumulation: 1
  mixed_precision: none
  unsloth: true
  lora_r: 16
  lora_alpha: 16
  lora_dropout: 0

hub:
  username: abc
  token: hf_XXX
  push_to_hub: false
```

### Error Logs

```
Loading checkpoint shards: 100% 2/2 [01:21<00:00, 40.56s/it]
INFO     | 2024-08-13 04:46:20 | autotrain.trainers.clm.utils:get_model:666 - model dtype: torch.float16
INFO     | 2024-08-13 04:46:20 | autotrain.trainers.clm.train_clm_sft:train:37 - creating trainer
/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_deprecation.py:100: FutureWarning: Deprecated argument(s) used in '__init__': dataset_text_field, max_seq_length, packing. Will not be supported from version '1.0.0'.

Deprecated positional argument(s) used in SFTTrainer, please use the SFTConfig to set these arguments instead.
  warnings.warn(message, FutureWarning)
/usr/local/lib/python3.10/dist-packages/trl/trainer/sft_trainer.py:192: UserWarning: You passed a `packing` argument to the SFTTrainer, the value you passed will override the one in the `SFTConfig`.
  warnings.warn(
/usr/local/lib/python3.10/dist-packages/trl/trainer/sft_trainer.py:280: UserWarning: You passed a `max_seq_length` argument to the SFTTrainer, the value you passed will override the one in the `SFTConfig`.
  warnings.warn(
/usr/local/lib/python3.10/dist-packages/trl/trainer/sft_trainer.py:318: UserWarning: You passed a `dataset_text_field` argument to the SFTTrainer, the value you passed will override the one in the `SFTConfig`.
  warnings.warn(
```

### Additional Information

I am not sure why this pops up. I know this is just a UserWarning and model is able to fine-tune ok, but is anything being affected?

**Comment by abhishekkrthakur** [comment]
Not really an error (atleast for now), so you can ignore it. it will be taken care of in next release 🙂

**Comment by jackswl** [comment]
thank you so much for the swift reply

**Comment by jackswl** [comment]
perhaps ill leave this reopened, until it has been taken care of in next release.

**Comment by abhishekkrthakur** [comment]
fixed in 0.8.9+ :)
